storey/writers.py

```diff
@@ -186,15 +211,19 @@ class WriteToTSDB(_Batching, _Writer):
     :param aggr_granularity: Granularity of server-side aggregations for this TSDB table (e.g. '1h').
     """
 
-    def __init__(self, path: str, time_col: str, columns: List[str], infer_columns_from_data: bool = False,
-                 index_cols: Union[str, List[str], None] = None, v3io_frames: Optional[str] = None, access_key: Optional[str] = None,
-                 container: str = "", rate: str = "", aggr: str = "", aggr_granularity: str = "", frames_client=None, **kwargs):
+    def __init__(self, path: str, time_col: str = '$time', columns: Union[str, List[str], None] = None,
+                 infer_columns_from_data: bool = False, index_cols: Union[str, List[str], None] = None, v3io_frames: Optional[str] = None,
```

Comment:
isn't `index_cols` better be `label_cols` in this specific step?

Reply:
We wanted consistency across writers, no? 🤷‍♂️